frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx

```diff
@@ -514,6 +523,13 @@ export default function UnitEditor(props: Props): JSX.Element {
     label: name
   }))
 
+  const employeeOptions = props.employees.map(
+    ({ id, firstName, lastName }) => ({
+      value: id,
+      label: [firstName, lastName].join(' ')
+    })
+  )
```

Comment:
Same thing here as in the filter component.

Reply:
To elaborate since the original comment isn't visible anymore, instead of doing this data processing every time this component renders, you could just do it once where you store the values in state. So do this processing in the part where you call `setEmployees`.